Scripts/assemble.sh

```diff
@@ -10,39 +10,39 @@ echo
 echo '\033[1mBuild: AblyAssetTrackingSubscriber Library\033[0m'
 echo
 
-xcodebuild -scheme "AblyAssetTrackingPublisher" -destination "generic/platform=iOS" SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES | xcpretty
+set -o pipefail & xcodebuild -scheme "AblyAssetTrackingPublisher" -destination "generic/platform=iOS" SKIP_INSTALL=NO BUILD_LIBRARY_FOR_DISTRIBUTION=YES | xcpretty
```

Comment:
Would it not be sufficient to put `set -o pipefail` once at the top of the file, below `set -e`?

Reply:
yes, once should be ok, I'll change it

Reply:
@lawrence-forooghian if t's ok, please resolve this conversation. Thanks

Reply:
👍